ynput/ayon-core
# Hiero: enhancing effect publishing categorization - AY-5146

## Changelog Description
This adds a setting and code for using track names when categorizing the soft effects in NukeStudio/Hiero.

## Testing notes:
1. In NukeStudio add effects to a track item and move those into separate video tracks.
2. In `ayon+settings://hiero/publish/CollectClipEffects/effect_tracks` add a category and track regex that match 1 or more track names.
3. Setup publishing of the track item and the effects should be categorized from the settings.
4. Publish and validate the effects are correct in Nuke.
